src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -110,7 +114,12 @@ private static void init(){}
         public static final IOptionalNamedTag<Block> ORES_NETHERITE_SCRAP = tag("ores/netherite_scrap");
         public static final IOptionalNamedTag<Block> ORES_QUARTZ = tag("ores/quartz");
         public static final IOptionalNamedTag<Block> ORES_REDSTONE = tag("ores/redstone");
-        public static final IOptionalNamedTag<Block> ORES_COPPER = tag("ores/copper");
+        public static final IOptionalNamedTag<Block> ORES_SINGULAR = tag("ores/singular");
```

Comment:
Singular, dense, and sparse really could used javadocs to explain them. The user should not be required to have read this PR to understand the distinction

Reply:
Indeed, I hadn't added them because I wasn't completely sure were to put them as none of the other tags had javadocs.

Reply:
I was thinking normal field javadocs. There are probably older tags that could use docs, but that's out of scope here